plugin/StrikeLightningConnectionStringHandler.cs

```diff
@@ -66,26 +70,29 @@ public StrikeLightningConnectionStringHandler(IServiceProvider serviceProvider,
 			}
 		}
 
-		if (!kv.TryGetValue("api-key", out var apiKey))
+		var convertToCurrency = Currency.Undefined;
+		if (kv.TryGetValue("convertto", out var convertToCurrencyStr))
```

Comment:
> convert-to? Similar to api-key.
Makes sense.

> We should consider renaming to receive-to because there is also a sending side where we can send payments from FIAT.
Fair point; but I do not have time now at this point to look into implementing and testing sending payments.

Reply:
`convert-to`: https://github.com/rockstardev/strike-btcpayserver-plugin/commit/4273ae814c077f6b3c611a9b08c932f79e421f6a